int16[2] point # pixel, in x y
---
geometry_msgs/Point result
